osu.Server.Spectator/Hubs/StatefulUserHub.cs

```diff
@@ -24,6 +26,9 @@ public abstract class StatefulUserHub<TClient, TUserState> : Hub<TClient>, ILogT
 
         private readonly Logger logger;
 
+        // ReSharper disable once StaticMemberInGenericType
```

Comment:
Open to a better solution if this is seen as something we don't want. Keep in mind that I intentionally want the `static` counter here to be local to each generic type.